docs/modules/ROOT/pages/contribute/index.adoc

```diff
@@ -1 +1,7 @@
-This page displays the information about how you can contribute to the documentation.
\ No newline at end of file
+= Contribute
+
+You can contribute to this documentation by selecting `Edit this Page`, beneath the search bar in the banner of these documents. This action takes you to the source docs on GitHub, where you can propose specific changes.
+
+When you have finished making your suggested changes in GitHub, select `Create a *new branch* for this commit and start a pull request`. Then, give your pull request a name and select `Propose changes.`
```

Comment:
Suggested edit: "When you finish editing in GitHub, select **Button name**. Then, name your pull request and..."